looker/looker-core-psc/main.tf

```
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Google LLC
```

Since this is a new sample, it should have the current date
```suggestion
 * Copyright 2025 Google LLC
```